### PR TITLE
[ID] Add "No Video" and "Target Practice"

### PR DESCRIPTION
**Pull request by wowcake**

### CODE REVIEW

File: wiki/Game_modifier/No_Video/id.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+![Ikon mod NV](/wiki/shared/mods/NV.png "Ikon mod No Video (NV)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+
+## Tentang
+
+- Singkatan: NV
+- Tipe: Spesial
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```
Comment (mousesalt) [inline wiki/Game_modifier/No_Video/id.md:23]:
```suggestion
- Mode-mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

File: wiki/Game_modifier/No_Video/id.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Deskripsi
+
+*Catatan: Mod ini tidak tersedia lagi untuk digunakan.*
+
+Mod **No Video** dahulunya adalah sebuah [game modifier](/wiki/Game_modifier) yang mengizinkan pemain untuk mematikan latar belakang video pada [beatmap](/wiki/Beatmap). Mod ini digunakan untuk meningkatkan framerate dan performa permainan.
+
+Fungsi mod ini telah digantikan dengan sebuah pengaturan pada bagian [Visual Settings](/wiki/Visual_Settings), karena itu, ikon dan mod ini tidak digunakan lagi.
```
Comment (mousesalt) [inline wiki/Game_modifier/No_Video/id.md:31]:
```suggestion
Fungsi mod ini telah digantikan dengan sebuah pengaturan pada bagian [Visual Settings](/wiki/Visual_Settings). Oleh sebab itu, ikon dan mod ini tidak digunakan lagi.
```

File: wiki/Game_modifier/Target_Practice/id.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Catatan: Mod Target Practice hanya dapat di akses menggunakan build `Cutting Edge`.*
+
+## Tentang
+
+- Singkatan: TP
+- Tipe: Spesial
+- Pengali Skor: 1.00x
+- Mode-mode permainan yang cocok: ![][osu!]
```
Comment (mousesalt) [inline wiki/Game_modifier/Target_Practice/id.md:23]:
```suggestion
- Mode permainan yang cocok: ![][osu!]
```

File: wiki/Game_modifier/Target_Practice/id.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## Deskripsi
+
+*Catatan: Menggunakan mod Target Practice akan mengakibatkan dalam sebuah unranked play.*
```
Comment (mousesalt) [inline wiki/Game_modifier/Target_Practice/id.md:27]:
```suggestion
*Catatan: Menggunakan mod Target Practice akan menjadikan unranked play.*
```

File: wiki/Game_modifier/No_Video/id.md (hunk 1 above)
Comment (wowcake) [reply wiki/Game_modifier/No_Video/id.md:23]: didn't we use ```game``` across the board?

Comment (mousesalt) [reply wiki/Game_modifier/No_Video/id.md:23]: we already discussed this for specific sub forum to using Mode Permainan instead

**Review by mousesalt — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**